src/libsync/creds/credentialmanager.cpp

```diff
@@ -80,15 +82,15 @@ QKeychain::Job *CredentialManager::set(const QString &key, const QVariant &data)
     connect(writeJob, &QKeychain::WritePasswordJob::finished, this, [writeJob, timer, key, timedOut = std::move(timedOut), this] {
         timer->stop();
         if (writeJob->error() == QKeychain::NoError) {
-            if (timedOut) {
-                qCInfo(lcCredentialsManager) << "set" << scopedKey(this, key) << "timed out";
+            if (*timedOut.get()) {
+                qCInfo(lcCredentialsManager) << "set" << writeJob->key() << "timed out";
             } else {
-                qCInfo(lcCredentialsManager) << "added" << scopedKey(this, key);
+                qCInfo(lcCredentialsManager) << "added" << writeJob->key() << key;
```

Comment:
Saying it was added is a bit of a bold claim, given that we have not verified this. Maybe rephrase to "assuming ... was added".